cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "body": {
+      "content": "delphi-getting-started.md",
+      "format": "md"
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "DELPHI software can be run directly from CVMFS. This works on various Linux flavors, and requires additional software to be installed. There is also a container image available, shipping with everything that is needed."
```

Comment:
Currently the search pages give: 

![gsd](https://github.com/cernopendata/opendata.cern.ch/assets/517546/5b0a2c8b-f7a0-4615-8229-1681f1276223)

It is perhaps not crucial to start with the fact that DELPHI software runs on CVMFS, because many visitors of the CERN Open Data portal would not know how to access it.

It's better to be as generic as possible in the short description, and explain all the details in the main body.

So, something like "This guide will explain you how to run the DELPHI software. Various flavours of the Linux operating systems are supported, as well as a possibility to run the software on CVMFS. Finally, the DELPHI software is also available as an encapsulated container image."

(just for illustration, you get my point about starting generic there)